virttest/utils_libvirt/libvirt_misc.py

```diff
@@ -0,0 +1,36 @@
+"""
+libvirt related misc functions
+"""
+import logging
+
+from avocado.core import exceptions
+
+from virttest import libvirt_version
+
+
+def is_libvirt_feature_supported(params, ignore_error=False):
+    """
+    Check whether the function is supported in this libvirt version by comparing
+    the installed libvirt version
+
+    :param params: Dictionary with the test parameters
+    :param ignore_error: Whether to raise an exception
+    :raise: When ignore_error is set False, raise TestCancel if the feature is
+        not supported
+    :return: True if the feature is supported;
+        False if the feature is not supported when ignore_error is set to True
+    """
+    func_supported_since_libvirt_ver = eval(
+        params.get("func_supported_since_libvirt_ver", '()'))
```

Comment:
Could you provide an example value for 'func_supported_since_libvirt_ver' in docstring for reference?

Reply:
Updated.